Resources/Locale/en-US/medical/components/defibrillator.ftl

```diff
@@ -1,4 +1,5 @@
-﻿defibrillator-not-on = The defibrillator isn't turned on.
+defibrillator-not-on = The defibrillator isn't turned on.
 defibrillator-no-mind = No intelligence pattern can be detected in patient's brain. Further attempts futile.
 defibrillator-rotten = Body decomposition detected: resuscitation failed.
 defibrillator-unrevivable = This patient is unable to be revived due to a unique body composition.
+defibrillator-xeno = Why would you want to bring this thing back!
```

Comment:
Why would you want to bring this thing back! -->  Why would you want to bring this thing back!?
 
 so its a question